### PR TITLE
Feature/stream engine support

### PR DESCRIPTION
**Pull request by gangtao**
- add engine support for stream
- add mutable stream test
- disable query result header check, some query has None header

### CODE REVIEW

File: python/timeplus/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "2.0.0b1"   # noqa: F401
+__version__ = "2.0.0b3"   # noqa: F401
```
Comment (jovezhong) [inline python/timeplus/version.py:1]: 2.0.0b3 has released yesterday, do you want to set this as b4?

**Review by ye11ow — APPROVED** [review]

File: python/timeplus/version.py (hunk 1 above)
Comment (gangtao) [reply python/timeplus/version.py:1]: this version will be overriden when run build automatically

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**